tree-sitter/tree-sitter.cabal

```diff
@@ -47,6 +47,7 @@ library
                      , TreeSitter.GenerateSyntax
                      , TreeSitter.Deserialize
                      , TreeSitter.Cursor
+  other-modules:       TreeSitter.Unmarshal.Examples
```

Comment:
We don’t want to actually expose these example datatypes to downstream consumers, so we include them in `other-modules` instead of `exposed-modules`.